server/lobbyconnection.py

```diff
@@ -819,7 +819,7 @@ def command_game_host(self, message):
 
         assert isinstance(self.player, Player)
 
-        title = cgi.escape(message.get('title', ''))
+        title = cgi.escape(message.get('title'), '')
```

Comment:
You still need to have the default of `''` in there if message doesn't have a `title` field. And if message does have a `title` field, but the value is `None`, you need to replace it with empty string. Also `cgi.escape` only takes one string parameter [https://docs.python.org/3/library/cgi.html#cgi.escape](https://docs.python.org/3/library/cgi.html#cgi.escape)

So it should look like this:

```suggestion
        title = cgi.escape(message.get('title', '') or '')
```

Which is pretty awful to read so maybe this is better:
```python
title = message.get('title', '') or ''
title = cgi.escape(title)
```

Or better yet, make a function for it.